### PR TITLE
Adding roadmap section to README.md

### PR DESCRIPTION
**Pull request by Sam-Burns**
This PR adds a 'roadmap' section to the readme.  This would be a good place to comment if there are any objections to the ideas suggested here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,6 +202,15 @@ $acclimator->registerAdapter('Your\Favorite\ContainerAdapter', 'Your\Favorite\Co
 $adaptedContainer = $acclimator->acclimate($container);
 ```
 
+## Roadmap
+
+Plans for Acclimate 2.0.0 include the following:
+
+- Adding support of the latest versions of all relevant containers.  This includes Laravel, Zend, Pimple and Symfony.
```
Comment (mnapoli) [inline README.md]: FYI the new version of Zend service manager already supports container-interop. For the rest, there is no need for a BC break as far as I know. Why create a new major version if that is just to drop PHP 5.4 support and Guzzle?

Comment (jeremeamia) [inline README.md]: That's a fair point. @Sam-Burns Maybe you should do a 1.1 with the Guzzle stuff marked as deprecated (which means we'll have to add it back in). Seems like the other updates have been backward-compatible so far.

**Comment by jeremeamia** [comment]
Also, did you start the [AcclimatePHP Twitter account](https://twitter.com/AcclimatePHP)? Nice. I think we should probably create an acclimate-specific GitHub org, so we can migrate this repo from my personal GitHub. @GrahamCampbell, any words of warning related to migrating repos as far as packagist/composer go? I feel like I've done this before and it worked fine, but I don't remember the details or steps.

**Comment by GrahamCampbell** [comment]
It should be fine because of the way GitHub redirects so long as you keep the package name the same,

**Comment by GrahamCampbell** [comment]
Just make sure you never fork the repo back to the old name so the redirect stays intact into the future for existing composer.locks.

**Comment by jeremeamia** [comment]
Would we need to update packagist somehow?

**Comment by jeremeamia** [comment]
@Sam-Burns what is your packagist username, so I can add you as a maintainer?
screen shot 2016-02-23 at 12 18 21 pm

**Comment by mnapoli** [comment]
By the way PHP-DI 3 is a pretty old version, I guess nobody uses it anymore (https://packagist.org/packages/php-di/php-di/stats#3.5.1). New versions are compatible with container-interop. So if you want you can remove it (it would be a BC break) or deprecate it.

**Comment by Sam-Burns** [comment]
@jeremeamia It's the same as my Github username.

**Comment by Sam-Burns** [comment]
I've updated the proposed 'roadmap' section of the docs to reflect everyone's feedback.  I'd like to move forward with version 1.1 quite quickly though - the last tagged release is 2 years and 38 commits old at this point, so it would be good to see version 1.1 released soon.

**Comment by Sam-Burns** [comment]
@jeremeamia And re the github org, it would be good to see the project moved with the redirect as you discussed.  I think you would have to do that, wouldn't you, as I only have push rights to the repo right now but aren't an admin?

**Comment by Sam-Burns** [comment]
@jeremeamia Do join us at https://github.com/AcclimateContainer and move the repo.  I'll take care of the rest.

**Comment by jeremeamia** [comment]
Hey, still want to merge this? Also, when should I move the repo?

**Comment by Sam-Burns** [comment]
@jeremeamia If you'd like to move the repo to the [Acclimate org](https://github.com/AcclimateContainer), that would be really good.  You're already an admin of the group I think :-)

**Comment by jeremeamia** [comment]
@Sam-Burns it's moved. Let's make sure to update anything else we need to update.

**Pull request closed without merging**